src/htsp_server.c

```diff
@@ -1083,13 +1083,34 @@ htsp_method_getSysTime(htsp_connection_t *htsp, htsmsg_t *in)
   htsmsg_t *out;
   struct timeval tv;
   struct timezone tz;
+  int tz_offset;
 
   if(gettimeofday(&tv, &tz) == -1)
     return htsp_error("Unable to get system time"); 
 
+  struct tm serverLocalTime;
+  tzset();
```

Comment:
Sorry, but my understanding is to call tzset only one time in app until the timezone (TZ variable) changes. It's called already in src/main.c and I don't see any benefit / requirement to call it more than one time in tvh.